pkgs/development/python-modules/gigalixir/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildPythonApplication
+, click
+, fetchPypi
+, git
+, httpretty
+, lib
+, qrcode
+, pygments
+, pyopenssl
+, pytest-runner
+, pytestCheckHook
+, requests
+, rollbar
+, stripe
+, sure
+}:
+
+buildPythonApplication rec {
+  pname = "gigalixir";
+  version = "1.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1b7a9aed7e61a3828f5a11774803edc39358e2ac463b3b5e52af267f3420dc66";
+  };
+
+  propagatedBuildInputs = [
+    click
+    requests
+    stripe
+    rollbar
+    pygments
+    qrcode
+    pyopenssl
+  ];
+
+  checkInputs = [
+    httpretty
+    sure
+    pytest-runner
```

Comment:
```suggestion
```
pytest-runner does not work and we patch it out.

Reply:
The package required it when tried to run without it, it would fail. Maybe I did something wrong?

Reply:
Done